packages/orbit-components/src/Inline/__tests__/index.test.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import React from "react";
+import { screen, render } from "@testing-library/react";
+
+import theme from "../../defaultTheme";
+import Inline from "..";
+
+const dataTest = "test";
+
+const tokens = {
+  none: "",
+  XXXsmall: theme.orbit.spaceXXXSmall,
+  XXsmall: theme.orbit.spaceXXSmall,
+  Xsmall: theme.orbit.spaceXSmall,
+  small: theme.orbit.spaceSmall,
+  medium: theme.orbit.spaceMedium,
+  large: theme.orbit.spaceLarge,
+  Xlarge: theme.orbit.spaceXLarge,
+  XXlarge: theme.orbit.spaceXXLarge,
+  XXXlarge: theme.orbit.spaceXXXLarge,
+};
+
+const Elements = () => (
+  <>
+    <div data-test="child-0">kek</div>
+    <div>bur</div>
+    <div>cheburek</div>
+    <div>blin</div>
+  </>
+);
+
+describe("#Inline", () => {
+  it("should have props", () => {
+    render(
+      <Inline align="center" justify="start" dataTest={dataTest}>
+        <Elements />
+      </Inline>,
+    );
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    const styles = screen.getByTestId(dataTest);
+    expect(styles).toHaveStyle({ alignItems: "center" });
+    expect(styles).toHaveStyle({ justifyContent: "flex-start" });
```

Comment:
The value of `styles` is a DOM element, so that name is a bit confusing. I recommend something like `container`:
```suggestion
    const container = screen.getByTestId(dataTest);
    expect(container).toHaveStyle({ alignItems: "center" });
    expect(container).toHaveStyle({ justifyContent: "flex-start" });
```